corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/content.less

```diff
@@ -101,5 +101,25 @@ body {
 }
 
 .flex-child-sidebar {
```

Comment:
It seems like this might be enough styling that it's worth breaking out into its own file, but I'm not sure when/whether that's appropriate.

Reply:
this is a bit outside my wheelhouse, but a less file just for case search sounds very reasonable to me